src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/PrisonerWithCvlFields.kt

```diff
@@ -42,13 +42,3 @@ data class PrisonerWithCvlFields(
   @field:Schema(description = "Additional CVL specific information including derived fields")
   val cvl: CvlFields,
 )
-
-@Schema(description = "A combination of the NOMIS prisoner record and their respective licence start date")
-data class CaseloadItem(
-  @field:Schema(description = "Details about a prisoner")
-  val prisoner: Prisoner,
-
-  @field:Schema(description = "Date that the licence is due to activate", example = "05/05/2023")
-  @field:JsonFormat(pattern = "dd/MM/yyyy")
-  val licenceStartDate: LocalDate? = null,
```

Comment:
Licence Start Date now comes from the CvlRecord, no need to fetch it as a part of a CaseloadService call, so this data class is redundant.